relnotes/delayed-suspender.feature.md

```diff
@@ -0,0 +1,18 @@
+## Fix `DelayedSuspender` for use with the request event dispatcher
+
+`swarm.neo.util.DelayedSuspender`
+
+Previously, using the `DelayedSuspender` with the request event dispatcher was
+buggy, as the suspended did not un/register the suspended fiber with the event
+dispatcher. This release adapts the code to allow the `DelayedSuspender` to be
+told about the existence of the request event dispatcher that's in use.
+
+Initialisation code like:
+```
+auto suspender = DelayedSuspender(fiber);
+```
+should be changed to:
+```
+auto suspender = DelayedSuspender(fiber, &request_event_dispatcher,
+    SuspendSignalCode);
```

Comment:
It's an optional change, though, in principle. (Though code that is not adapted will be buggy.)

Reply:
Yeah, I'm scared of that _buggy_ bit. I wouldn't insist on it, though.

Reply:
The alternative would be to do this as a `+breaking` patch release. I suppose that's the strictly correct way.

Reply:
Yes, that would be the strictly correct way to do this (this is what I had in mind with my first comment, sorry for not being clear for the patch release).

Reply:
Even cooler would be if you can make the change so that you _have_ to modify the code to use it or else compilation fails. No hidden-buggy behavior then.

Reply:
Changing the order of the fields in the structure would work.

Reply:
Ah you're right. I can just change the order of the fields :+1:

Reply:
Updated.